CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changes by Version
 ==================
 
-next release (yyyy-mm-dd)
```

Comment:
We'll want to keep the next release section, esp. for documenting breaking changes.

Reply:
Addressed in last push